controller/controller_test.go

```diff
@@ -318,12 +318,12 @@ func (s *TestSuite) TestIsSameGuaranteedCPURequirement(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(IsSameGuaranteedCPURequirement(a, b), Equals, true)
 
-	b.Requests[corev1.ResourceCPU], err = resource.ParseQuantity("250m")
+	b.Requests[corev1.ResourceCPU], _ = resource.ParseQuantity("250m")
```

Comment:
Use `c.Assert(err, IsNil)` rather than ignoring it in the unit tests. Similar below.